setup.py

```diff
@@ -134,6 +134,52 @@ def gen_packages_items():
 def get_extensions():
     extensions = []
 
+    if os.getenv('MMCV_WITH_TRT', '0') != '0':
+        ext_name = 'mmcv._ext_trt'
+        from torch.utils.cpp_extension import include_paths, library_paths
+        library_dirs = []
+        libraries = []
+        include_dirs = []
+        ort_path = os.getenv('TENSORRT_DIR', '0')
```

Comment:
The name of some variables should be changed to a meaningful one. Such as `ort_path`, `ort_lib_path`